bench-tps/src/send_batch.rs

```diff
@@ -109,17 +109,35 @@ pub fn generate_durable_nonce_accounts<T: 'static + BenchTpsClient + Send + Sync
     let (mut nonce_keypairs, _extra) = generate_keypairs(seed_keypair, count as u64);
     nonce_keypairs.truncate(count);
 
-    let to_fund: Vec<(&Keypair, &Keypair)> = authority_keypairs
+    let to_fund: Vec<NonceCreateSigners> = authority_keypairs
```

Comment:
I replace pair to struct in order to have two different types for Withdraw and Create accounts. It is required because for Create I need two signatures and for Withdraw only one.